devices/YarpRobotLoggerDevice/scripts/blf-logger-with-audio.sh

```diff
@@ -1,6 +1,32 @@
 #!/usr/bin/env bash
 
-input_port_audio=$1
+# Parse command line arguments
+while [[ $# -gt 0 ]]
+do
+    key="$1"
+
+    case $key in
+        -i|--input-port-audio)
+        input_port_audio="$2"
+        shift
+        shift
+        ;;
+        -l|--launch-file)
+        launch_file="$2"
+        shift
+        shift
+        ;;
+        *)
+        # Unknown option
+        echo "Unknown option: $1"
+        exit 1
+        ;;
+    esac
```

Comment:
Could you add some comments about the available options?

Reply:
Just to make myself clearer, I meant to specify which inputs are parsed, like
```
Available options:
-i, --input-port-audio, name of the yarp port to connect in order to log audio
-l, --launch-file, name of the file to launch for the logger yarprobotinterface
```
or something similar